src/architect/cluster/cluster.utils.ts

```diff
@@ -14,11 +14,11 @@ export interface CreateClusterInput {
 export type ClusterCredentials = KubernetesClusterCredentials;
 
 export interface KubernetesClusterCredentials {
-  kind: 'KUBERNETES';
+  kind: 'KUBERNETES' | 'AGENT';
 
   host: string;
-  cluster_ca_cert: string;
-  service_token: string;
+  cluster_ca_cert?: string;
```

Comment:
Instead of making these optional, should there be a separate `AgentClusterCredentials`?

Reply:
Honestly I was debating removing them all together since they are no longer used, but figured this was okay for now.